README.md

```diff
@@ -1,45 +1,55 @@
 # AIP++
 
-The full documentation of our application can be found at 
-https://cutt.ly/Mn78T3f
-### Running the application locally using Docker
+> A Software Tool for Finding Related Work in Academia
 
+<ins>A</ins>rticle <ins>I</ins>nformation <ins>P</ins>arser is an instrument to parse, unify, and in some cases correct article metadata. AIP creates a PostgreSQL database that allows for easily finding related work. AIP++ is an extension of the original tool to improve on the article discovery and querying capabilities of the original tool.
 
-1. Make sure you have Docker installed and running on your machine.
+More information on how this tool works can be found in the [project report](https://drive.google.com/file/d/1OUJvakiJYdlw-jV2ZKjDs5ggCNi2DyiW/view).
```

Comment:
Please donwload this pdf and put it in the repo, that link might be broken one day.

Reply:
Done :)